twiter/logic.js

```diff
@@ -0,0 +1,70 @@
+const Tweeter = function () {
+
+    const posts = [{ id: 1, txt: "I am the First", comments: [], commentIdCounter: 0 }]
+    let postIdCounter = 1
+
+    const getPosts = function () {
+        return posts
+    }
+
+
+    const getComments = function (postID,) {
+        let go = getPosts()
+        for (let i = 0; i < go.length; i++) {
+            if (go[i].id == postID) {
+                let com = go[i].comments
+                return com
+
+            }
+        }
+    }
+
+
+    const addPost = function (text) {
+        postIdCounter++
+        let somePost = { id: postIdCounter, txt: text, comments: [], commentIdCounter: 0 }
+        posts.push(somePost)
+    }
+
+    const removePost = function (postID) {
+        for (let i = 0; i < posts.length; i++) {
+            if (postID == posts[i].id) {
+                posts.splice(i, 1)
+            }
+        }
+    }
+
+    const addComment = function (postId, text) {
+        for (let i = 0; i < posts.length; i++) {
+            if (postId == posts[i].id) {
+                let com = posts[i].comments
```

Comment:
event though `com` is short, short is not always better. It is really important to have simple readable code. Sometimes writing out a bit more makes your code a lot more easy to follow and more professional. 

`const newComment = ` would be better